pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -113,6 +113,17 @@ type PrebindPlugin interface {
 	Prebind(pc *PluginContext, p *v1.Pod, nodeName string) *Status
 }
 
+// PostbindPlugin is an interface that must be implemented by "postbind" plugins.
+// These plugins are called after a pod was scheduled with a cycle.
```

Comment:
This line should be:
"These plugins are called after a pod is successfully bound to a node."